packages/nimble-blazor/NimbleBlazor/Components/NimbleRadioGroup.razor

```diff
@@ -0,0 +1,11 @@
+@namespace NimbleBlazor
+@inherits NimbleInputBase<string>
+<nimble-radio-group disabled=@Disabled
+                    name="@Name"
+                    value="@Value"
+                    orientation="@Orientation.ToAttributeValue()"
+                    current-value="@CurrentValue"
```

Comment:
Same comment regarding setting `value` and `current-value`.

Reply:
Removed value